source/leuven/wice_advanced_guide.rst

```diff
@@ -0,0 +1,121 @@
+:ref:`overview <software_development>`.
+
+.. _wice_worker:
+
+Worker
+------
+
+The :ref:`worker framework <worker framework>`, which allows to conveniently
+parameterize simulations, is available on wICE. An attention point is that
+if you want to lauch worker jobs from the Genius login nodes, you will need to
+use a specific module:
+
+::
+
+    $ module use /apps/leuven/skylake/2021a/modules/all
+    $ module load worker/1.6.12-foss-2021a-wice
+
+If instead you want to launch Worker jobs from an interactive job running on
+wICE, you can use the ``worker/1.6.12-foss-2021a`` module (but do make sure
+this is the version installed in a subdirectory of ``/apps/leuven/icelake``).
+
+.. _wice_monitoring:
+
+Monitoring
+----------
+
+For monitoring or debugging jobs on wICE, you can look into tools provided by
+Slurm such as:
+
+* `scontrol <https://slurm.schedmd.com/scontrol.html>`__ to view Slurm
+  configuration and state
+* `squeue <https://slurm.schedmd.com/squeue.html>`__ to get information about
+  jobs in the scheduling queue
+* `sacct <https://slurm.schedmd.com/sacct.html>`__ to display information about
+  finished jobs
+
+For convenience, we provide the ``slurm_jobinfo`` tool, which runs and parses
+output from the Slurm tools mentioned above into a format that is easier to
+read. Simply use ``slurm_jobinfo <jobid>`` where ``<jobid>`` has to be replaced
+by the 8-digit number that identifies your job.
+
+For getting a compact overview of the current state of the cluster, execute
+``slurmtop`` on any KU Leuven Tier-2 node. Use ``slurmtop --help`` to get to
+know the functionality.
+
+.. _wice_known_issues:
+
+Known issues
+------------
+
+Intel MPI pinning
+=================
+
+The Intel MPI library does not always play well with the Slurm scheduler on
+wICE. Specifically, when launching a job from a compute node (for instance from
+inside an interactive job), processes are not pinned correctly. This issue can
+be overcome by setting the environment variable ``I_MPI_PIN_RESPECT_CPUSET=off``
+or equivalently adding the option ``-env I_MPI_PIN_RESPECT_CPUSET=off`` to your
+mpirun command. To check that processes are pinned correctly to physical cores,
+set the environment variable ``I_MPI_DEBUG=5`` to get more verbose output. Note
+that this issue does not occur with the Open MPI library.
+
+Environment propagation
+=======================
+
+Jobs on wICE do not start with a clean slate. Some information from the session
+in which the job was submitted is propagated. For instance modules that were
+loaded at job submission time, will also be loaded inside the job. This can
+create problems when you submit a job to wICE from a Genius login node, because
+modules installed for genius are generally not suited to be used on wICE nodes.
+You can make sure this does not happen by executing ``module --force purge`` at
+the start of your job script, which will unload any currently loaded module.
```

Comment:
Furthermore, any `export`ed environment variable on your Genius login session will be propagated to your job on wICE. Therefore, you must avoid exporting critical environment variables such as `PATH` and/or `LD_LIBRARY_PATH` from your `.bashrc` file (or by other means). Otherwise, you will fall into module inconsistencies and code misbehavior on wICE.

(or any other text similar to the suggestion above)